components/scream/src/share/scream_logging.hpp

```diff
@@ -0,0 +1,271 @@
+
+#ifndef SCREAM_LOGGING_HPP
+#define SCREAM_LOGGING_HPP
+
+#include <cstdio>
+#include <cstdlib>
+#include <string>
+#include <vector>
```

Comment:
There are a lot of includes here, some not necessary (like vector). I'm a big fan of 'include only what you need/use'.